### DISCUSSION TITLE
Error with aiocsv

### DISCUSSION THREAD

**Discussion by Silent-EXE4** [ExpressApp/pybotx#477] [Q&A]
При установке Pybotx выдает ошибки с aiocsv: 
![image](https://github.com/ExpressApp/pybotx/assets/174069837/d531bc26-8a5c-4186-a520-3c8e8da04145)

Если установить aiocsv отдельно, то начинает ругаться на несовместимость версий. Нужную версию установить не получается:
![image](https://github.com/ExpressApp/pybotx/assets/174069837/a74ed7e7-e53d-435c-b59a-0b03150cec71)

Подскажите пожалуйста есть ли какое либо решение данной проблемы?

**Comment by osovskiyalexandr** [comment]
Добрый день! Можно было устанвоить aiocsv, до версии 1.3.0. Однако, данная проблема скорее уже не актуальна и вам, и версию мы подняли - в pybotx)
